src/client/requests/globals.rs

```diff
@@ -0,0 +1,27 @@
+use graphql_client::reqwest::post_graphql;
+
+use crate::{
+    client::{
+        queries::{query_globals, QueryGlobals, QueryGlobalsGlobals},
+        GaloyClient,
+    },
+    errors::{api_error::ApiError, CliError},
+};
+
+impl GaloyClient {
+    pub async fn globals(&self) -> Result<QueryGlobalsGlobals, CliError> {
+        let variables = query_globals::Variables;
+
+        let response_body =
+            post_graphql::<QueryGlobals, _>(&self.graphql_client, &self.api, variables)
+                .await
+                .map_err(|_| ApiError::IssueGettingResponse)?;
```

Comment:
I wouldn't eat the previous error - then you never find out what the issue might be. You can wrap it if you want but don't just drop it.

Reply:
Addressed this in a106ff98bbe1579b58865e8ac62717b0fafca644